security.md

```diff
@@ -66,67 +76,91 @@ Deprecated, do not use.
+
+See [w3c-ccg](https://w3c-ccg.github.io/security-vocab/#expires)
 
 <h4 id="initializationVector"><a href="#initializationVector">initializationVector</a></h4>
 
-Deprecated, do not use.
+The initialization vector (IV) is a byte stream that is typically used to initialize certain block cipher encryption schemes. For a receiving application to be able to decrypt a message, it must know the decryption key and the initialization vector. The value is typically base-64 encoded.
+
+See [w3c-ccg](https://w3c-ccg.github.io/security-vocab/#initializationVector)
 
 <h4 id="iterationCount"><a href="#iterationCount">iterationCount</a></h4>
 
 Deprecated, do not use.
 
 <h4 id="nonce"><a href="#nonce">nonce</a></h4>
 
-Deprecated, do not use.
+This property is used in conjunction with the input to the signature hashing function in order to protect against replay attacks. Typically, receivers need to track all nonce values used within a certain time period in order to ensure that an attacker cannot merely re-send a compromised packet in order to execute a privileged request.
+
+See [w3c-ccg](https://w3c-ccg.github.io/security-vocab/#nonce)
 
 <h4 id="normalizationAlgorithm"><a href="#normalizationAlgorithm">normalizationAlgorithm</a></h4>
 
 Deprecated, do not use.
 
-<h4 id="owner"><a href="#owner">owner</a></h4>
+Also known as canonicalizationAlgorithm.
 
-Deprecated, do not use.
+See [w3c-ccg](https://w3c-ccg.github.io/security-vocab/#canonicalizationAlgorithm)
 
 <h4 id="password"><a href="#password">password</a></h4>
 
-Deprecated, do not use.
+A secret that is used to generate a key that can be used to encrypt or decrypt message. It is typically a string value.
+
+See [w3c-ccg](https://w3c-ccg.github.io/security-vocab/#password)
 
 <h4 id="privateKey"><a href="#privateKey">privateKey</a></h4>
 
 Deprecated, do not use.
 
 <h4 id="privateKeyPem"><a href="#privateKeyPem">privateKeyPem</a></h4>
 
-Deprecated, do not use.
+A private key PEM property is used to specify the PEM-encoded version of the private key. This encoding is compatible with almost every Secure Sockets Layer library implementation and typically plugs directly into functions intializing private keys.
+
+You should use privateKeyJwk when possible.
+
+See [w3c-ccg](https://w3c-ccg.github.io/security-vocab/#privateKeyPem).
 
 <h4 id="publicKey"><a href="#publicKey">publicKey</a></h4>
 
-Deprecated, do not use.
+A public key property is used to specify a URL that contains information about a public key.
+
+See [w3c-ccg](https://w3c-ccg.github.io/security-vocab/#publicKey).
 
 <h4 id="publicKeyBase58"><a href="#publicKeyBase58">publicKeyBase58</a></h4>
 
-Deprecated, do not use.
+You should use publicKeyJwk if possible.
```

Comment:
This statement should either be motivated by a list of reasons, or removed

Reply:
The reason is that publickKeyJwk is valid for all supported key formats according to the did core spec. I will update.